core-services/image-mirroring/hypershift/mapping_hypershift_latest

```diff
@@ -0,0 +1 @@
+registry.ci.openshift.org/hypershift/hypershift-operator:latest quay.io/hypershift/hypershift:latest
```

Comment:
should be `quay.io/hypershift/hypershift-operator:latest` no?

Reply:
yes, nice catch